Makefile

```diff
@@ -1,18 +1,21 @@
 run.jarbas:
-	docker-compose up -d --build
-	docker-compose run --rm jarbas python manage.py ceapdatasets
-	docker-compose run --rm jarbas python manage.py migrate
+	docker-compose -f docker-compose-dev.yml up -d --build
+	docker exec jarbas_jarbas python manage.py ceapdatasets
+	docker exec jarbas_jarbas python manage.py migrate
 
 
 collectstatic: run.jarbas
-	docker-compose run --rm jarbas python manage.py collectstatic --no-input
+	docker exec jarbas_jarbas python manage.py collectstatic --no-input
 
 seed: run.jarbas
-	docker-compose run --rm jarbas python manage.py reimbursements /tmp/serenata-data/reimbursements.xz
-	docker-compose run --rm jarbas python manage.py companies /tmp/serenata-data/2016-09-03-companies.xz
-	docker-compose run --rm jarbas python manage.py irregularities /tmp/serenata-data/irregularities.xz
+	docker exec jarbas_jarbas python manage.py reimbursements /tmp/serenata-data/reimbursements.xz
+	docker exec jarbas_jarbas python manage.py companies /tmp/serenata-data/2016-09-03-companies.xz
+	docker exec jarbas_jarbas python manage.py irregularities /tmp/serenata-data/irregularities.xz
```

Comment:
Using docker-compose exec we need to specify the docker-compose-(dev/prod).yml used in the command. It can be painful even with makefile because the docker-compose-(dev/prod).yml will be hardcoded and will need to change. With docker binary we don't need to worry about that, even in dev or prod environment.